client/src/main/scala/sbt/client/impl/DebugClient.scala

```diff
@@ -13,7 +13,7 @@ private[sbt] object DebugClient {
     val uuid = java.util.UUID.randomUUID()
     val configName = "debug-client"
     val humanReadableName = "Debug Client"
-    client.sendJson[Message](RegisterClientRequest(ClientInfo(uuid.toString, configName, humanReadableName)),
+    client.sendJson[Message](RegisterClientRequest(ClientInfo(uuid.toString, configName, humanReadableName, "1.0", Vector.empty)),
```

Comment:
Just rename the directory for saved protocol too and any version is ok, even the name of your favorite marsupial